modules/ensemble/lib/widget/widget_registry.dart

```diff
@@ -51,25 +48,28 @@ import 'package:ensemble/widget/markdown.dart';
 import 'package:ensemble/widget/popup_menu.dart';
 import 'package:ensemble/widget/progress_indicator.dart';
 import 'package:ensemble/widget/qr_code.dart';
+import 'package:ensemble/widget/radio/radio_button.dart';
+import 'package:ensemble/widget/radio/radio_group.dart';
 import 'package:ensemble/widget/rating.dart';
+import 'package:ensemble/widget/shape.dart';
 import 'package:ensemble/widget/signature.dart';
+import 'package:ensemble/widget/slidable.dart';
 import 'package:ensemble/widget/spacer.dart';
 import 'package:ensemble/widget/staggered_grid.dart';
+import 'package:ensemble/widget/static_map.dart';
 import 'package:ensemble/widget/stub_widgets.dart';
 import 'package:ensemble/widget/switch.dart';
 import 'package:ensemble/widget/text.dart';
 import 'package:ensemble/widget/toggle_button.dart';
 import 'package:ensemble/widget/tooltip.dart';
 import 'package:ensemble/widget/video.dart';
-import 'package:ensemble/widget/slidable.dart';
-import 'package:ensemble/widget/accordion.dart';
 import 'package:ensemble/widget/visualization/barchart.dart';
 import 'package:ensemble/widget/visualization/chart_js/chart_js.dart';
 import 'package:ensemble/widget/visualization/line_area_chart.dart';
 import 'package:ensemble/widget/visualization/topology_chart.dart';
 import 'package:ensemble/widget/webview/webview.dart';
-import 'package:ensemble/widget/youtube/youtube.dart';
 import 'package:ensemble/widget/weeklyscheduler.dart';
+import 'package:ensemble/widget/youtube/youtube.dart';
```

Comment:
also i dont see any required change in this file, please update the changes if not required.